waggle-dance-integration-tests/src/main/java/com/hotels/bdp/waggledance/WaggleDanceRunner.java

```diff
@@ -133,6 +139,25 @@ public Builder federate(String name, String remoteMetaStoreUris, AccessControlTy
       return this;
     }
 
+    public Builder federateWithMetastoreTunnel(
```

Comment:
Why do we need this? Is it used anywhere other than a test?

Reply:
It is just for the integration test, but I could not find any other way of making the integration test include something that takes metastore-tunnel options too.

Reply:
This is testing that the configuration can be loaded correctly. When I first tried to replace MetastoreTunnel with SshSettings, the bean couldn't be initialised. On a real environment, the change was failing, but the integration tests were all passing successfully. Where do you think it would be better to test this behaviour?
However, the test does pass when I run it on my IDE, but I wonder why...

Reply:
discussed offline, but we decided to keep it as it is something that benefits from testing :)

Reply:
OK, good to know. I'm going to "resolve conversation" then!